src/test/java/com/wikia/webdriver/common/core/elemnt/Wait.java

```diff
@@ -212,16 +221,24 @@ public WebElement forElementVisible(By by) {
     }
   }
 
-  public WebElement forElementVisible(By by, int timeout, int polling) {
+  /**
+   * @deprecated use method with Duration object except int
+   */
+  @Deprecated
+  public WebElement forElementVisible(By by, int timeoutSec, int polling) {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1133)